### DISCUSSION TITLE
"Never Researched" option for scenarios

### DISCUSSION THREAD

**Discussion by MaxMallon** [OpenRCT2/OpenRCT2#16455] [Feature requests]
It would be cool if it were possible to include scenery and rides in scenarios that are not available during scenario play. Scenery used to make the outside-the-park area doesn't nessesarily need to be available inside the park, for example. It could also give interesting dynamics with rides. Imagine having a certain prebuilt rollercoaster, that if you were to destroy it, you can't ever get back. Or for example, having only one cash machine be available in total. Things like that would be fun, but are currently impossible.
So I suggest having a third option in inventions list: never researched.

**Comment by IntelOrca** [comment]
This is kind of already implemented now in `develop`, but there is no UI for controlling it. If you open a SC4 park like Forest Frontiers or Bumbly Beach, you will notice that scenery outside the park is no longer available to place just like in RCT1.

**Reply by MaxMallon** [reply]
Ooh that is really cool! Then only a UI needs to be added!

Does it work for rides too though?

**Reply by duncanspumpkin** [reply]
Yes it would work for both.

**Reply by MaxMallon** [reply]
Is it possible to add stuff like that using the debugger perhaps?

**Comment by J-PIE-314** [comment]
The request for restricted scenery has been implemented in PR #21455. A ride restriction UI has not been implemented yet.
